user-interface/src/lib/components/uswds/date-range-picker.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface DateRangePickerRef {
```

Comment:
Since we are doing the same thing on each input field, would it be better to move all these refs into a single file? like input-fields.d.ts. 

Unless we think we will be expanding on these individually I think we should combine them

Reply:
More than that, we could actually use the same interface for each of the refs, if they are all the same.  Then later if we ever need something more specific for one of these components, then we can create one specific for *that* component.

Reply:
Let's discuss